examples/docs/src/pages/features/latex.mdx

```diff
@@ -1,3 +1,7 @@
+import {MathJaxContext} from "better-react-mathjax"
+
+<MathJaxContext>
```

Comment:
Tip to wrap your mdx file with this component you can go to mdx2 playground put this code and see how ast nodes should looks like, if you’ll not succeed I’ll help with it

Reply:
I assume I cannot just wrap the whole page since it may have `import` statements at the beginning? Do you have an example where just the body is wrapped in a component?